cfgov/v1/tests/test_filterable_list.py

```diff
@@ -1,107 +1,74 @@
-import os
 import mock
 
-from django import forms
 from django.test import TestCase
-from django.core.exceptions import ValidationError
-from django.core.paginator import EmptyPage, PageNotAnInteger
 from django.test import RequestFactory
 
-from ..models import BrowseFilterablePage, EventArchivePage, NewsroomLandingPage, SublandingFilterablePage, ActivityLogPage
 from ..util.filterable_list import FilterableListMixin
+from v1.forms import FilterableListForm
 
 
 class TestFilterableListMixin(TestCase):
     def setUp(self):
         self.mixin = FilterableListMixin()
         self.factory = RequestFactory()
 
-    # FilterableListMixin.get_form_class tests
-    def test_get_form_class_returns_form_class(self):
-        assert issubclass(self.mixin.get_form_class(), forms.Form)
-
-
-    # FilterableListMixin.get_page_set tests
     @mock.patch('v1.models.learn_page.AbstractFilterPage.objects')
-    def test_get_page_set_calls_live_shared(self, mock_cfgovmanager):
-        form = mock.Mock()
-        hostname = 'hostname'
-        self.mixin.get_page_set(form, hostname)
+    def test_base_query_calls_live_shared(self, mock_cfgovmanager):
+        FilterableListForm(parent=None, hostname='hostname').base_query()
         assert mock_cfgovmanager.live_shared.called
```

Comment:
@chosak Good point, Andy.  I updated the tests I added in `test_filterable_list_form`.